assets/source/sass/subsites/visitystad/_slider.scss

```diff
@@ -0,0 +1,14 @@
+.slider {
+    .text-block {
+        color: $simrishamn-darker-gray;
+        background-color: rgba($header-bg, 0.8) !important;
+    }
+
+    .flickity-page-dots {
+        .dot {
+            &.is-selected {
```

Comment:
`.dot.is-selected { ... }` ?

Reply:
Might want to add styling to `.dot` in the future.